docs/undergrad.yaml

```diff
@@ -414,7 +414,7 @@ Topology:
     contraction mapping theorem: 'contracting_with.exists_fixed_point'
   Normed vector spaces on $\R$ and $\C$:
     topology on a normed vector space: 'normed_space.topological_vector_space'
-    equivalent norms:
+    equivalent norms: 'linear_equiv.to_continuous_linear_equiv_of_bounds'
```

Comment:
I don't think this is correct. Here we want the definition of the equivalence relation on the type of norms on a given vector space given by N_1 \leq CN_2 and N_2 \leq C'N_1.

Reply:
A couple of lines later we have
```
equivalence of norms in finite dimension: 'linear_equiv.to_continuous_linear_equiv'
```
is that also too weak a statement?

Reply:
This is a theorem and not a definition so we have more freedom for interpretation.

Reply:
```suggestion
    equivalent norms: 
```